app/scripts/lib/strings.js

```diff
@@ -20,6 +20,8 @@ function () {
 
   t('Sign up to learn more');
 
+  t('Firefox is coming soon to iOS!');
```

Comment:
we will not, in fact, both this string and the string on line 21 should be removed if they are both in the template.